frontend/packages/console-plugin-sdk/src/store.ts

```diff
@@ -134,9 +138,16 @@ export class PluginStore {
   }
 
   private updateExtensions() {
-    this.dynamicPluginExtensions = Array.from(this.loadedDynamicPlugins.values()).reduce(
+    const dynamicPlugins = Array.from(this.loadedDynamicPlugins.values());
+
+    this.staticPluginExtensions = this.staticPlugins.reduce((acc, plugin) => {
+      const equivalentDynamicPlugin = dynamicPlugins.find((p) => p.manifest.name === plugin.name);
+      return equivalentDynamicPlugin?.enabled ? acc : [...acc, ...plugin.extensions];
+    }, [] as LoadedExtension[]);
```

Comment:
@vojtechszocs @spadgett 
I think we want to ignore static plugins regardless of whether or not the dynamic plugin is enabled; so long as the dynamic plugin is present.

Reply:
I think this is probably right. Otherwise you could think you're using the dynamic plugin since the UI is there, but it's not really enabled.

My worry is that users won't know they need to enable the plugin after updating their operator, and the UI simply disappears. We were trying to address that in https://github.com/openshift/console-operator/pull/576, but it requires us to know what plugins to look for.

I feel like we need some way to let users know that a plugin is available after upgrading an operator. Right now it's only apparent when installing new operators.

Reply:
I'll do the change as suggested by @christianvogt above.